Gemfile.lock

```diff
@@ -11,7 +11,7 @@ PATH
       railties
       rainbow
       thor (>= 0.18.1, < 2.0)
-      webpush (~> 1.0)
+      web-push
```

Comment:
[webpush](https://github.com/zaru/webpush) is not updating for 4 years. [web-push](https://github.com/pushpad/web-push) seems to have better maintainability.